### DISCUSSION TITLE
Claim Blocks

### DISCUSSION THREAD

**Discussion by GoaTTheGoat** [GriefPrevention/GriefPrevention#1178] [Q&A] [answered]
What is the command to give claimblocks to players?

**Comment by Jikoo** [comment, marked as answer]
Did you check [the wiki](https://github.com/TechFortress/GriefPrevention/wiki/Commands#admin-commands)?

**Reply by GoaTTheGoat** [reply]
> Did you check [the wiki](https://github.com/TechFortress/GriefPrevention/wiki/Commands#admin-commands)?

Yes, but I can't seem to find what command is to give players claimblocks. It's not the claimblocks per hour.

**Reply by Jikoo** [reply]
![image](https://user-images.githubusercontent.com/1353270/103254018-aedf9900-4951-11eb-9413-72ca94452382.png)
